### PR TITLE
Add hyper.sh to production users

### PR DESCRIPTION
**Pull request by gnawux**
It would be our pleasure to be listed as a production user

### CODE REVIEW

**Comment by xiang90** [comment]
@gnawux Can you change to the commit message to

doc: Add hyper.sh to production users?

Thanks.

**Comment by gnawux** [comment]
@xiang90 updated

File: Documentation/production-users.md [hunk 1]
```diff
@@ -130,3 +130,17 @@ PD(Placement Driver) is the central controller in the TiDB cluster. It saves the
...
+
+- *Application*: Kubernetes, distributed locks, etc.
+- *Launched*: April 2016
+- *Cluster Size*: 1 cluster of 3 members
+- *Order of Data Size*: 10s of MB
+- *Operator*: Hyper.sh
+- *Environment*: Baremetal
+- *Backups*: None, all data can be recreated if necessary.
+
+[hypernetes](https://github.com/hyperhq/hypernetes)
```
Comment (xiang90) [inline Documentation/production-users.md]: can you put a few sentences to explain your use case here? not just like these 3 key words? Thanks!

Comment (gnawux) [inline Documentation/production-users.md]: updated.

File: Documentation/production-users.md [hunk 2]
```diff
@@ -130,3 +130,19 @@ PD(Placement Driver) is the central controller in the TiDB cluster. It saves the
...
+- *Launched*: April 2016
+- *Cluster Size*: 1 cluster of 3 members
+- *Order of Data Size*: 10s of MB
+- *Operator*: Hyper.sh
+- *Environment*: Baremetal
+- *Backups*: None, all data can be recreated if necessary.
+
+In hyper.sh, the container service is backed by hypernetes, a multi-tenant kubernetes distro. Moreover, we use etcd to coordinate the multiple manage services and store global meta data.
+
+[hypernetes](https://github.com/hyperhq/hypernetes)
```
Comment (xiang90) [inline Documentation/production-users.md]: this renders incorrectly. you might want to do 
```

In hyper.sh, the container service is backed by [hypernetes][hypernetes]

[hypernetes] : https://github.com/hyperhq/hypernetes
```

Comment (gnawux) [inline Documentation/production-users.md]: sorry for the links, updated

**Comment by xiang90** [comment]
LGTM. Thanks.

**Pull request merged**